### PR TITLE
SAK-48407 Profile: Takes more steps to update a profile pic

### PR DESCRIPTION
**Pull request by hornersa**
The save button needed a data-bs-dismiss to dismiss the modal when saving the change. I also renamed the 'Done' button to 'Cancel'. I found the meaning of the former to be confusing, especially in relation to 'Save'. The button actually cancels without preserving changes made in the modal.

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/js/sui-picture-changer/sui-picture-changer.js [hunk 1]
```diff
@@ -253,9 +253,9 @@ export class SuiPictureChanger extends SakaiElement {
             </div>
         </div>
         <div class="modal-footer">
-          <button class="btn pull-left" @click=${this._save} ?disabled=${!this.needsSave}>${this.i18n.save}</button>
+          <button class="btn pull-left" data-bs-dismiss="modal" @click=${this._save} ?disabled=${!this.needsSave}>${this.i18n.save}</button>
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/sui-picture-changer/sui-picture-changer.js:256]:
```suggestion
          <button class="btn float-start" data-bs-dismiss="modal" @click=${this._save} ?disabled=${!this.needsSave}>${this.i18n.save}</button>
```

File: webcomponents/tool/src/main/frontend/js/sui-picture-changer/sui-picture-changer.js [hunk 2]
```diff
@@ -253,9 +253,9 @@ export class SuiPictureChanger extends SakaiElement {
...
           <button class="btn pull-right" @click="${this._remove}">${this.i18n.remove}</button>
-          <button class="btn pull-left" data-bs-dismiss="modal">${this.i18n.done}</button>
+          <button class="btn pull-left" data-bs-dismiss="modal">${this.i18n.cancel}</button>
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/sui-picture-changer/sui-picture-changer.js:258]:
```suggestion
          <button class="btn float-start" data-bs-dismiss="modal">${this.i18n.cancel}</button>
```

**Review by adrianfish — CHANGES_REQUESTED** [review]
Thanks Sean. Just a couple of changes with bs3's pull-left -> bs5's float-start.

**Comment by hornersa** [comment]
@adrianfish - I've implemented the suggested changes.

File: webcomponents/tool/src/main/frontend/js/sui-picture-changer/sui-picture-changer.js [hunk 3]
```diff
@@ -253,9 +253,9 @@ export class SuiPictureChanger extends SakaiElement {
             </div>
         </div>
         <div class="modal-footer">
-          <button class="btn pull-left" @click=${this._save} ?disabled=${!this.needsSave}>${this.i18n.save}</button>
-          <button class="btn pull-right" @click="${this._remove}">${this.i18n.remove}</button>
-          <button class="btn pull-left" data-bs-dismiss="modal">${this.i18n.done}</button>
+          <button class="btn float-start" data-bs-dismiss="modal" @click=${this._save} ?disabled=${!this.needsSave}>${this.i18n.save}</button>
+          <button class="btn float-start" @click="${this._remove}">${this.i18n.remove}</button>
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/sui-picture-changer/sui-picture-changer.js:257]:
```suggestion
          <button class="btn float-end" @click="${this._remove}">${this.i18n.remove}</button>
```

**Pull request merged**